### PR TITLE
Issue #795 Note on having VT-x in prerequisite

### LINKED ISSUE #795: Latest version Minishift RC 2 not working in fresh installation of Virtualbox
Hi, 

I am using latest version which was released yesterday in Fedora 25. I am getting some kernel related issues.

My Virtualbox version : `5.1.20 rc..`

```
[skumar@dhcp-0-118 ~]$ ./minishift version
Minishift version: 1.0.0-rc.2
[skumar@dhcp-0-118 ~]$ 
[skumar@dhcp-0-118 ~]$ 
[skumar@dhcp-0-118 ~]$ ./minishift start --vm-driver virtualbox
Starting local OpenShift cluster using 'virtualbox' hypervisor...
Downloading ISO 'https://github.com/minishift/minishift-b2d-iso/releases/download/v1.0.2/minishift-b2d.iso'
 40.00 MB / 40.00 MB [=================================================================================================================================================================================] 100.00% 0s
E0421 08:57:34.603601   15935 start.go:171] Error starting the VM: Error creating the VM. Error with pre-create check: "We support Virtualbox starting with version 5. Your VirtualBox install is \"WARNING: The vboxdrv kernel module is not loaded. Either there is no module\\n         available for the current kernel (4.10.8-200.fc25.x86_64) or it failed to\\n         load. Please recompile the kernel module and install it by\\n\\n           sudo /sbin/vboxconfig\\n\\n         You will not be able to start VMs until this problem is fixed.\\n5.1.20r114628\". Please upgrade at https://www.virtualbox.org". Retrying.
E0421 08:57:36.655704   15935 start.go:171] Error starting the VM: Error creating the VM. Error with pre-create check: "We support Virtualbox starting with version 5. Your VirtualBox install is \"WARNING: The vboxdrv kernel module is not loaded. Either there is no module\\n         available for the current kernel (4.10.8-200.fc25.x86_64) or it failed to\\n         load. Please recompile the kernel module and install it by\\n\\n           sudo /sbin/vboxconfig\\n\\n         You will not be able to start VMs until this problem is fixed.\\n5.1.20r114628\". Please upgrade at https://www.virtualbox.org". Retrying.
E0421 08:57:38.700887   15935 start.go:171] Error starting the VM: Error creating the VM. Error with pre-create check: "We support Virtualbox starting with version 5. Your VirtualBox install is \"WARNING: The vboxdrv kernel module is not loaded. Either there is no module\\n         available for the current kernel (4.10.8-200.fc25.x86_64) or it failed to\\n         load. Please recompile the kernel module and install it by\\n\\n           sudo /sbin/vboxconfig\\n\\n         You will not be able to start VMs until this problem is fixed.\\n5.1.20r114628\". Please upgrade at https://www.virtualbox.org". Retrying.
Error starting the VM:  Error creating the VM. Error with pre-create check: "We support Virtualbox starting with version 5. Your VirtualBox install is \"WARNING: The vboxdrv kernel module is not loaded. Either there is no module\\n         available for the current kernel (4.10.8-200.fc25.x86_64) or it failed to\\n         load. Please recompile the kernel module and install it by\\n\\n           sudo /sbin/vboxconfig\\n\\n         You will not be able to start VMs until this problem is fixed.\\n5.1.20r114628\". Please upgrade at https://www.virtualbox.org"
Error creating the VM. Error with pre-create check: "We support Virtualbox starting with version 5. Your VirtualBox install is \"WARNING: The vboxdrv kernel module is not loaded. Either there is no module\\n         available for the current kernel (4.10.8-200.fc25.x86_64) or it failed to\\n         load. Please recompile the kernel module and install it by\\n\\n           sudo /sbin/vboxconfig\\n\\n         You will not be able to start VMs until this problem is fixed.\\n5.1.20r114628\". Please upgrade at https://www.virtualbox.org"
Error creating the VM. Error with pre-create check: "We support Virtualbox starting with version 5. Your VirtualBox install is \"WARNING: The vboxdrv kernel module is not loaded. Either there is no module\\n         available for the current kernel (4.10.8-200.fc25.x86_64) or it failed to\\n         load. Please recompile the kernel module and install it by\\n\\n           sudo /sbin/vboxconfig\\n\\n         You will not be able to start VMs until this problem is fixed.\\n5.1.20r114628\". Please upgrade at https://www.virtualbox.org"

```

### PR DESCRIPTION
**Pull request by budhrg**
Fix #795

Also reorder the way 'Getting Started' displays as per overview.

### CODE REVIEW

File: docs/source/_topic_map.yml [hunk 1]
```diff
@@ -10,12 +10,12 @@ Topics:
...
       - Name: Overview
         File: index
-      - Name: Installing Docker Machine drivers
-        File: docker-machine-drivers
       - Name: Installing Minishift
         File: installing
       - Name: Quickstart
         File: quickstart
+      - Name: Installing Docker Machine drivers
+        File: docker-machine-drivers
```
Comment (LalatenduMohanty) [inline docs/source/_topic_map.yml]: @budhrg User should first go through docker machine driver documentation then should go through minishift installation docs. The original sequence is right IMO. Not sure why you changed it.

File: docs/source/getting-started/installing.adoc [hunk 2]
```diff
@@ -16,8 +16,8 @@ The following section describes how to install Minishift and the required depend
 == Prerequisites
 
 Minishift requires a hypervisor to start the virtual machine on which the OpenShift cluster
-is provisioned. Make sure that the hypervisor of your choice is installed and enabled on
-your system before you start Minishift.
+is provisioned. Ensure you have Virtualization Technology (VT-x) enabled in your system and
```
Comment (thatdocslady) [inline docs/source/getting-started/installing.adoc]: Change: Ensure you have <virt tech>... > Make sure that <virt tech> is enabled on your system

File: docs/source/getting-started/installing.adoc [hunk 3]
```diff
@@ -16,8 +16,8 @@ The following section describes how to install Minishift and the required depend
...
+the hypervisor of your choice is installed and working before you start Minishift.
```
Comment (thatdocslady) [inline docs/source/getting-started/installing.adoc]: and > and that

change "working" to "enabled"/"running" (working is too ambiguous)

Comment (LalatenduMohanty) [inline docs/source/getting-started/installing.adoc]: I think we do not need to document this as users are supposed to have a working hypervisor setup. For that they should refer respective hypervisor documentations. I mean we have to draw a line some where about what we want to document and what we do not want to.

**Comment by budhrg** [comment]
Closing this PR as we are assuming that user will have their hypervisors installed through respective hypervisor site which includes this documentation.

**Pull request closed without merging**